### ISSUE TITLE
Deprecation warning about yaml.load in distributed.config

### ISSUE THREAD

**Issue by vvegorova** [dask/distributed#2564]
There is a `DeprecationWarning` raised by `distributed.config`:
```
/opt/conda/envs/name/lib/python3.6/site-packages/dask/distributed.py:5: in <module>
    from distributed import *
/opt/conda/envs/name/lib/python3.6/site-packages/distributed/__init__.py:3: in <module>
    from . import config
/opt/conda/envs/name/lib/python3.6/site-packages/distributed/config.py:20: in <module>
    defaults = yaml.load(f)
/opt/conda/envs/name/lib/python3.6/site-packages/yaml/__init__.py:109: in load
    load_warning('load')
/opt/conda/envs/name/lib/python3.6/site-packages/yaml/__init__.py:55: in load_warning
    warnings.warn(message, YAMLLoadWarning, stacklevel=3)
E   yaml.YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
```

A new version of `pyyaml` released on Mar 13, 2019 deprecates the usage of `yaml.load(input)`:
https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation.

**Comment by TomAugspurger** [comment]
Thanks for the report. Are you interested in submitting a PR to update our loading? I don't think we need any of the unsafe features.

**Comment by vvegorova** [comment]
Yes, I'd like to do it. Should [dask.config](https://github.com/dask/dask/blob/master/dask/config.py) be updated as well?

**Comment by mrocklin** [comment]
See also https://github.com/dask/dask/pull/4603

**Comment by ayushdg** [comment]
I believe this should be resolved with #2566

**Comment by mrocklin** [comment]
Indeed.  Thanks for the reminder @ayushdg !  Closing.
